### PR TITLE
Update install-manual.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

Update install-manual.mdx

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Review by at-the-vr — COMMENTED** [review]
will it be a good thing to specify `max-width` so that the button is not stretched out
| Instance | Button Look |
| -------- | ------- |
| Old Docs | image |
| New Docs | image |

File: src/content/docs/en/install/manual.mdx [hunk 1]
```diff
@@ -12,7 +12,9 @@ This guide will walk you through the steps to manually install and configure a n
 
 #### Prefer a quicker way to get started?
 
+<p>
 <Button href="/en/install/auto/">Try the create astro CLI wizard →</Button>
+</p>
```
Comment (delucis) [inline src/content/docs/en/install/manual.mdx:17]: Thanks for spotting this @liruifengv! I wonder if actually the button treatment can be dropped now in favour of something more like what we have on https://docs.astro.build/en/install/auto/ ?

```suggestion
:::tip[Prefer a quicker way to get started?]
Follow the [create astro CLI wizard](/en/install/auto/) instead.
:::
```

Comment (liruifengv) [reply src/content/docs/en/install/manual.mdx:17]: It's great!

**Comment by sarah11918** [comment]
Just chiming in that we probably want to rework some text here:

Do this if you prefer not to use the CLI tool.
Prefer to use the CLI tool???

😆 

image

File: src/content/docs/en/install/manual.mdx [hunk 2]
```diff
@@ -10,9 +10,9 @@ import ReadMore from '~/components/ReadMore.astro'
 
 This guide will walk you through the steps to manually install and configure a new Astro project if you prefer not to use [the automatic CLI tool](/en/install/auto/).
```
Comment (sarah11918) [inline src/content/docs/en/install/manual.mdx:11]:
```suggestion
This guide will walk you through the steps to manually install and configure a new Astro project.
```
Let the tip do the talking?

**Review by sarah11918 — APPROVED** [review]
Looks great! Thank you for making this way better. 💜

**Comment by sarah11918** [comment]
This one needs:

Co-authored-by: Sarah Rainsberger <5098874+sarah11918@users.noreply.github.com>
Co-authored-by: Chris Swithinbank <357379+delucis@users.noreply.github.com>
Co-authored-by: Atharva <88548999+at-the-vr@users.noreply.github.com>

**Pull request merged**